ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py

```diff
@@ -399,6 +400,50 @@ def test_paddle_expand(
     )
 
 
+# expand_as
+@st.composite
+def _expand_as_helper(draw):
+    dtype_and_x = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+            min_num_dims=1,
+            max_num_dims=6,
+        )
+    )
+
+    dtype, x = dtype_and_x
+    input_shape = x[0].shape
+
+    max_num_dims = 6 - len(input_shape)
+    shape = draw(helpers.get_shape(max_num_dims=max_num_dims)) + input_shape
+
+    return dtype, x, shape
+
+
+@st.composite
+def dtype_x_and_shape(draw):
+    input_dtype = draw(st.sampled_from(["float32", "int32", "float64"]))
+    shape = draw(st.lists(st.integers(min_value=1, max_value=10), min_size=1, max_size=4))
+    x = draw(helpers.array_values(shape=shape, dtype=input_dtype))
+    return input_dtype, x, shape
+
+@given(dtype_x_and_shape())
```

Comment:
the decorator used here should be `@handle_frontend_test`, kindly refer to our docs for a better understanding.